src/pages/CreateEvent/ChallengeDetails.tsx

```diff
@@ -50,40 +59,48 @@ const ChallengeDetailsForm: React.FC = () => {
       </div>
 
       <div className="form-group">
-        <label htmlFor="challengeReleaseDate">Challenge Release Date*</label>
-        <Controller
-          name="challengeReleaseDate"
-          control={control}
-          rules={{ required: "Challenge release date is required" }}
-          render={({ field }) => (
-            <div className="date-input-container">
-              <CalendarIcon className="icon" />
+        <label htmlFor="challengeReleaseDate">Challenge Release Date *</label>
+        <div className="date-input-container">
+          <Controller
```

Comment:
You can restrict it so the organizer publishes the event at least 2 weeks before the event start date

Reply:
Done